encrypted-media-respec.html

```diff
@@ -1604,15 +1611,95 @@ <h2><dfn>MediaKeys</dfn> Interface</h2>
           </ol>
         <table class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><td class="prmName">sessionType</td><td class="prmType"><code>MediaKeySessionType = "temporary"</code></td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptTrue"><span role="img" aria-label="True">✔</span></td><td class="prmDesc">
               The type of session to create. The session type affects the behavior of the returned object.
-            </td></tr></tbody></table><div><em>Return type: </em><code>MediaKeySession</code></div></dd><dt><dfn><code>setServerCertificate</code></dfn></dt><dd>
+            </td></tr></tbody></table><div><em>Return type: </em><code>MediaKeySession</code></div></dd>
+            <dt><dfn><code>getStatusForPolicy</code></dfn></dt>
+            <dd>
+              <p id="status-for-policy">Returns the <a>MediaKeyStatus</a> for a given <a>MediaKeysPolicy</a>.</p>
+
+              <ol class="method-algorithm">
+                <li>If <var>policy</var> has no <a def-id="present-dictionary-member"></a> <a def-id="dictionary-member">members</a>, return a promise rejected with a newly created <a def-id="TypeError"></a>.</li>
```

Comment:
I don't know how the links work in the spec. But there seems to be some issue with the link on "present" and "members", based on the preview page.

The first one points to https://webidl.spec.whatwg.org/#map-exists, which seems to be a broken anchor in the page.

The second one points to https://webidl.spec.whatwg.org/##dfn-dictionary-member, which should be https://webidl.spec.whatwg.org/#dfn-dictionary-member (## -> #).